ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -240,11 +240,15 @@ - (NSDictionary *) getDeviceNamesByCode {
         @"iPhone12,1": @"iPhone 11",
         @"iPhone12,3": @"iPhone 11 Pro",
         @"iPhone12,5": @"iPhone 11 Pro Max",
+	@"iPhone12,8": @"iPhone SE", // (2nd Generation iPhone SE),
         @"iPhone13,1": @"iPhone 12 mini",
         @"iPhone13,2": @"iPhone 12",
         @"iPhone13,3": @"iPhone 12 Pro",
         @"iPhone13,4": @"iPhone 12 Pro Max",
-        @"iPhone12,8": @"iPhone SE", // (2nd Generation iPhone SE)
+	@"iPhone14,1": @"iPhone 13 mini",
+        @"iPhone14,2": @"iPhone 13",
+        @"iPhone14,3": @"iPhone 13 Pro",
+        @"iPhone14,4": @"iPhone 13 Pro Max",
```

Comment:
Hi @rolfb 
for iPhone 13 the above combination is not working , whereas for other devices of 13 series its working as expected , i tried with following combinations and looks me fine , i m not sure what is the number game of 1,2,34 vs 5,2,3,4 , sorry i am not IOS expert , just done some experiment with this numbers 
        @"iPhone14,5": @"iPhone 13 mini",
        @"iPhone14,2": @"iPhone 13",
        @"iPhone14,3": @"iPhone 13 Pro",
        @"iPhone14,4": @"iPhone 13 Pro Max",

Reply:
@hiteshSavkare Great feedback, how exactly are you testing this?

Reply:
@rolfb Thanks,
 react-native-device-info into is integrated in my project, so changed the file RNDeviceInfo.m with above code  and building the project with xcode 13 and testing on simulators of  13 series

Reply:
@hiteshSavkare updated the file now, going to try to verify with the most recent xcode beta as well.

Reply:
Sounds good @rolfb,  thanks +1

Reply:
Great @rolfb , Hope you tested with simulators or Do i need to test again ?

Reply:
@hiteshSavkare didn't check as I expect Xcode's own lookup tables to be correct. Feel free to verify as the list from Xcode isn't exactly matching the one you provided.

Reply:
@rolfb looks fine :+1: